src/scripts/genLttngProvider.py

```diff
@@ -555,8 +555,10 @@ def generateLttngFiles(etwmanifest,eventprovider_directory):
 
         tracepointprovider_Cmake.write("""    )
 
+        find_library(LTTNG NAMES lttng-ust)
+
         target_link_libraries(coreclrtraceptprovider
-                              -llttng-ust
+                             ${LTTNG}
```

Comment:
A micro-nit - there is one space missing in front of the `${LTTNG}` so that it is aligned with the coreclrtraceptprovider string above.